bindings/typescript/src/election.ts

```diff
@@ -0,0 +1,56 @@
+import { ElementModP, ElementModQ } from "./group";
+import { CiphertextElectionContextHandle, getInstance } from "./wasm";
+
+export class ElectionContext {
```

Comment:
most of these are just stubs for now allowing serialization to/from json and not much else yet.